doc/src/manual/interacting-with-julia.md

```diff
@@ -172,6 +172,7 @@ to do so).
 | `^Y`                | "Yank" insert the text from the kill buffer                                      |
 | `^T`                | Transpose the characters about the cursor                                        |
 | `^Q`                | Write a number in REPL and press `^Q` to open editor at corresponding stackframe or method |
+| `^/`, `^_`          | Undo |
```

Comment:
I didn't see you add a key for `^/`, how does it work? (when I tried to rebase last year, I had compilation problems with this key...)
Also, your commit message refers to "Ctrl-^", you should change it.